app/resources/v1/users_pg.js

```diff
@@ -10,7 +10,7 @@ const Op = Sequelize.Op
 
 exports.post = async function (req, res) {
   let loginToken = null
-  const hasPriorCall = !!(res.mainData && res.mainData.id)
+  const hasPriorCall = false
```

Comment:
If this is always false and isn't based on anything anymore, why keep this variable?